app/(main)/components/Hero.tsx

```
@@ -59,17 +54,27 @@ export function Hero({ data }: { data: FetchedData }) {
+                className="absolute inset-0 min-h-full w-screen object-cover tablet:h-full"
+              >
+                <source src="/HomeVideoBackground.mp4" type="video/mp4" />
+              </video>
+            ) : (
+              <img
+                src="/SkyBackgroundLowRes.png"
+                alt="Sky background"
+                className="absolute inset-0 min-h-full w-screen object-cover tablet:h-full"
+              />
+            )}
+          </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)